# PatchService messages
patch.service.not_relevant=Non relevante para o esquema {0}
patch.executer.checking=Buscando parches para aplicar...
patch.executer.no_patches_required=Non foi necesario ning\u00FAn parche.
patch.executer.system_readonly=Non \u00E9 posible aplicar parches a un sistema s\u00F3 de lectura. Pode ser que existan incompatibilidades entre o c\u00F3digo da aplicaci\u00F3n e os datos existentes.
patch.executer.not_executed =\n\
=== Parche gravado (non executado) === \n\
ID: {0}\n\
RESULTADO: \n\
{1}\n\
=====================================
patch.executer.executed     =\n\
=== Parche aplicado                 === \n\
ID: {0}\n\
RESULTADO: \n\
{1}\n\
=====================================
patch.executer.failed       =\n\
=== Erro ao aplicar o parche         === \n\
ID: {0}\n\
RESULTADO: \n\
{1}\n\
=====================================

# General patch messages

patch.genericBootstrap.result.exists=A localizaci\u00F3n de reinicio xa existe: {0}
patch.genericBootstrap.result.created=Visualizaci\u00F3n importada \u00E1 localizaci\u00F3n de reinicio: {0} ({1})
patch.genericBootstrap.err.multiple_found=Localiz\u00E1ronse varios n\u00F3s: {0}

patch.general.property_not_set=Non se definiu a propiedade '{0}'' neste parche: {1}

# Individual patch messages

patch.marker.description=Parche de marcador para gravar instalaci\u00F3ns e actualizaci\u00F3ns
patch.marker.result=Parche de marcador aplicado

patch.savedSearchesFolder.description=Garante a existencia do cartafol Buscas gardadas.
patch.savedSearchesFolder.result.exists=O cartafol Buscar gardadas xa existe: {0}
patch.savedSearchesFolder.result.created=Cartafol Buscas gardadas creado con \u00E9xito: {0}

patch.savedSearchesPermission.description=Define os permisos necesarios no cartafol Buscas gardadas.
patch.savedSearchesPermission.result.applied=Concedido o papel de CONTRIBUTOR a EVERYONE no cartafol Buscas gardadas: {0}.
patch.savedSearchesPermission.err.not_found=Non foi posible localizar o cartafol Buscas gardadas.

patch.updatePermissionData.description=Actualizar permisos de 'folder' a 'cmobject' [JIRA: AR-344].
patch.updatePermissionData.result={0} entradas de control de acceso 'folder' modificadas para 'cmobject'.

patch.authoritiesFolder.description=Garante a existencia do cartafol de autoridades de usuario [JIRA: AR-497].

patch.authoritiesFolderPermission.description=Garante que as autoridades de grupo estean visibles para todos.

patch.guestUser.description=Engadir o usuario convidado e o espazo principal do convidado e repara os permisos do espazo principal da empresa, do convidado e da persoa convidada. 
patch.guestUser.result=Usuario convidado engadido e permisos reparados.

patch.fixNodeSerializableValues.description=Garantir que os valores de propiedade non estean almacenados como Serializable se \u00E9 posible
patch.fixNodeSerializableValues.result=Reparados {0} valores serializados da propiedade do n\u00F3

patch.updateGuestPermission.description=Renomear o permiso de convidado de Convidado para Consumidor
patch.updateGuestPermission.result=Mud\u00E1ronse {0} entradas de control de acceso de Convidado para Consumidor.

patch.categoryRootPermission.description=Define os permisos necesarios do cartafol Ra\u00EDz da categor\u00EDa.
patch.categoryRootPermission.result=Concedido o papel de CONSUMER a GUEST no cartafol Ra\u00EDz da categor\u00EDa: {0}.
patch.categoryRootPermission.err.not_found=Non foi posible localizar o cartafol Ra\u00EDz da categor\u00EDa ({0}).

patch.guestPersonPermission.description=Modificar o permiso de Persoa convidada de Consumidor a Lector
patch.guestPersonPermission.result=Permiso de Persoa convidada actualizado de Consumidor a Lector

patch.spacesRootPermission.description=Modificar o permiso ra\u00EDz do Almac\u00E9n de espazos de Consumidor a Lector
patch.spacesRootPermission.result=Permiso ra\u00EDz do Almac\u00E9n de espazos actualizado de Consumidor a Lector

patch.contentPermission.description=Actualizar entradas de permiso de 'cm:content' para 'sys:base'.
patch.contentPermission.result={0} entradas de control de acceso 'cm:content' modificadas para 'sys:base'.

patch.forumsIcons.description=Actualiza as referencias de icona dos foros
patch.forumsIcons.result={0} referencias de icona actualizadas

patch.emailTemplatesFolder.description=Garante a existencia do cartafol Modelos de correo electr\u00F3nico.
patch.emailTemplatesFolder.result.exists=O cartafol Modelos de correo electr\u00F3nico xa existe: {0}
patch.emailTemplatesFolder.result.created=Cartafol Modelos de correo electr\u00F3nico creado con \u00E9xito: {0}

patch.emailTemplatesContent.description=Carga os modelos no cartafol Modelos de correo electr\u00F3nico.
patch.emailTemplatesContent.result=Import\u00E1ronse os Modelos de correo electr\u00F3nico no cartafol predefinido.

patch.descriptorUpdate.description=Actualizar o descritor do repositorio
patch.descriptorUpdate.result=Descritor de repositorio actualizado

patch.scriptsFolder.description=Garante a existencia do cartafol Secuencias de comando.
patch.scriptsFolder.result.exists=O cartafol Secuencias de comando xa existe: {0}
patch.scriptsFolder.result.created=Cartafol Secuencias de comando creado con \u00E9xito: {0}

patch.topLevelGroupParentChildAssociationTypePatch.description=Garantir que os grupos de nivel superior te\u00F1an o tipo correcto de asociaci\u00F3n secundaria.
patch.topLevelGroupParentChildAssociationTypePatch.result=Repar\u00E1ronse {0} tipos de asociaci\u00F3n secundaria dos grupos de nivel superior.
patch.topLevelGroupParentChildAssociationTypePatch.err.sys_path_not_found=Non se localizou o cami\u00F1o de sistema da autoridade necesario: {0}
patch.topLevelGroupParentChildAssociationTypePatch.err.auth_path_not_found=Non se localizou o cami\u00F1o de autoridade necesario: {0}

patch.actionRuleDecouplingPatch.description=Migrar as regras existentes ao modelo actualizado, en que ser\u00E1n desligadas das acci\u00F3ns.
patch.actionRuleDecouplingPatch.result={0} regras actualizadas.

patch.systemWorkflowFolder.description=Garante a existencia do contedor de fluxo de traballo do sistema.
patch.systemWorkflowFolder.result.created=Contedor de fluxo de traballo do sistema creado {0}.

patch.rssTemplatesFolder.description=Garante a existencia do cartafol Modelos RSS.
patch.rssTemplatesFolder.result.exists=O cartafol Modelos RSS xa existe: {0}
patch.rssTemplatesFolder.result.created=Cartafol Modelos RSS creado con \u00E9xito: {0}

patch.uifacetsAspectRemovalPatch.description=Elimina o aspecto uifacets aplicado incorrectamente dos ficheiros do modelo de presentaci\u00F3n.
patch.uifacetsAspectRemovalPatch.updated=O aspecto uifacets eliminouse con \u00E9xito de {0} ficheiros do modelo de presentaci\u00F3n.

patch.guestPersonPermission2.description=Modificar o permiso de Persoa convidada a visible por todos os usuarios como Consumidor.
patch.guestPersonPermission2.result=Permiso de Persoa convidada actualizada a visible por todos os usuarios como Consumidor.

patch.schemaUpgradeScript.description=Garante a execuci\u00F3n da secuencia de comandos da actualizaci\u00F3n da base de datos.
patch.schemaUpgradeScript.err.not_executed=A secuencia de comandos da actualizaci\u00F3n do esquema, ''{0}'', non se executou nesta base de datos.

patch.uniqueChildName.description=Verifica e renomea os elementos secundarios duplicados.
patch.uniqueChildName.copyOf=({0}-{1})
patch.uniqueChildName.result={0} asociaci\u00F3ns verificadas e {1} duplicados reparados.  Consulte o ficheiro {2} para obter detalles.
patch.uniqueChildName.err.unable_to_fix=Erro ao reparar automaticamente os nomes duplicados. Consulte o ficheiro {0} para obter detalles.

patch.invalidNameEnding.description=Repara os finais dos nomes cun espazo ou punto.
patch.invalidNameEnding.result={0} finais de nomes reparados cun espazo ou punto. Consulte o ficheiro {1} para obter detalles.
patch.invalidNameEnding.err.unable_to_fix=Erro ao reparar automaticamente os finais dos nomes cun espazo ou punto. Consulte o ficheiro {0} para obter detalles.
patch.invalidNameEnding.rewritten=Nome ''{0}'' reescrito como ''{1}''

patch.systemDescriptorContent.description=Engade o contido das propiedades da versi\u00F3n ao descritor do sistema.
patch.systemDescriptorContent.result=Engadiuse o contido das propiedades da versi\u00F3n ao descritor do sistema.
patch.systemDescriptorContent.err.no_version_properties=Non foi posible localizar o recurso version.properties.
patch.systemDescriptorContent.err.no_descriptor=Non foi posible localizar o descritor do sistema.

patch.versionHistoryPerformance.description=Mellora o rendemento das buscas do historial de versi\u00F3ns.
patch.versionHistoryPerformance.result=Actualiz\u00E1ronse {0} obxectos do historial de versi\u00F3ns para mellorar o rendemento.

patch.multilingualBootstrap.description=Reinicia o n\u00F3 que manter\u00E1 os contedores multiling\u00FCes.

patch.wcmFolders.description=Garante a existencia dos cartafoles Proxectos web e Formularios web espec\u00EDficos do WCM.
patch.wcmFolders.webprojects.result.exists=O cartafol Proxectos web xa existe: {0}
patch.wcmFolders.webprojects.result.created=Cartafol Proxectos web creado con \u00E9xito: {0}
patch.wcmFolders.webforms.result.exists=O cartafol Formularios web xa existe: {0}
patch.wcmFolders.webforms.result.created=Cartafol Formularios web creado con \u00E9xito: {0}

patch.linkNodeExtension.description=Repara as extensi\u00F3ns de ficheiro do n\u00F3 da ligaz\u00F3n para que te\u00F1an a extensi\u00F3n .url.
patch.linkNodeExtension.result=Reparadas {0} extensi\u00F3ns. Consulte o ficheiro {1} para obter detalles.
patch.linkNodeExtension.err.unable_to_fix=Erro ao reparar automaticamente as extensi\u00F3ns de ficheiro do n\u00F3 da ligaz\u00F3n. Consulte o ficheiro {0} para obter detalles.
patch.linkNodeExtension.rewritten=Nome ''{0}'' reescrito como ''{1}''

patch.systemRegistryBootstrap.description=Reinicia o n\u00F3 que manter\u00E1 os metadatos do rexistro do sistema.

patch.userAndPersonUserNamesAsIdentifiers.description=Indexar de novo os identificadores de usuario usr:user e cm:person uids como identificadores
patch.userAndPersonUserNamesAsIdentifiers.result=Indexados os identificadores de usuario user:user e cm:person uids como identificadores

patch.contentFormFolderType.description=Actualizar o tipo de cartafol do Formulario de contido  WCM.
patch.contentFormFolderType.result=Actualizados {0} obxectos do Formulario de contido WCM para o tipo wcm:formfolder.

patch.groupNamesAsIdentifiers.description=Indexar de novo os gids usr:authorityContainer como identificadores
patch.groupNamesAsIdentifiers.result=Indexados usr:authorityContainer con identificadores

patch.invalidUserPersonAndGroup.description=Reparar uids non v\u00E1lidos para persoas e usuarios e gids para grupos
patch.invalidUserPersonAndGroup.result=Reparados ''{0}'' n\u00F3s de usuario non v\u00E1lidos, ''{1}'' n\u00F3s de persoa non v\u00E1lidos e ''{2}'' n\u00F3s de autoridade non v\u00E1lidos.

patch.AVMGuidPatch.description=Definir GUIDs como n\u00F3s AVM.
patch.AVMGuidPatch.result=GUIDS de AVM definidos.
 
patch.webscripts.description=Engade secuencias de comando web ao Dicionario de datos.
patch.webscriptsExtension.description=Engade a extensi\u00F3n das secuencias de comando web ao Dicionario de datos.

patch.AVMLayeredSnapshot.description=Define indirectionVersion en n\u00F3s con capas.
patch.AVMLayeredSnapshot.result=indirectionVersions do n\u00F3 con capas definido.

patch.groupMembersAsIdentifiers.description=Indexar de novo os membros usr:authorityContainer como identificadores=======

patch.genericWorkflow.result.deployed=Redespregados {0} fluxos de traballo. 

patch.redeploySubmitProcess.description=Redespregar Definici\u00F3n do proceso de env\u00EDo de WCM.

patch.AVMLocking.description=Engade os proxectos web existentes ao servizo de bloqueo.
patch.AVMLocking.result=Engadidos os proxectos web necesarios.

patch.AVMAspects.description=Modifica a almacenaxe de aspectos en n\u00F3s AVM.
patch.AVMAspects.result=Mov\u00E9ronse os aspectos.

patch.ReadmeTemplate.description=Modelo L\u00E9ame despregado
patch.webScriptsReadme.description=Aplicouse o modelo L\u00E9ame aos cartafoles de secuencias de comandos web

patch.AVMProperties.description=Modifica a almacenaxe de propiedades en n\u00F3s AVM.
patch.AVMProperties.result=Mov\u00E9ronse as propiedades.

